release-prepare.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+VERSION="$1"
+
+# update remote base reference tag in examples
+yq -i ". | .resources[0] |= sub(\"^(.+ref=)(\\d+\\.\\d+\\.\\d+)$\", \"\${1}${VERSION}\")" kustomize/overlays/remote-examples/namespace/kustomization.yaml
```

Comment:
Cool! some extensive scripting knowledge you have @JoergSiebahn :)

Reply:
I'm just trained putting the right keywords in search engines 🙈